github/ci/prow-deploy/files/jobs/kubevirt/libguestfs-appliance/libguestfs-appliance-presubmits.yaml

```diff
@@ -0,0 +1,29 @@
+presubmits:
+  kubevirt/libguestfs-appliance:
+  - name: check-build-libguestfs-appliance
+    always_run: true
+    optional: false
+    decorate: true
+    decoration_config:
+      timeout: 3h
+    max_concurrency: 1
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror: "true"
+    cluster: phx-prow
```

Comment:
We don't need the bare-metal cluster. You can place here `cluster: ibm-prow-jobs`

Reply:
thanks, I was not sure which one to pick :)